ui/dashboards/src/components/DashboardToolbar.tsx

```diff
@@ -27,7 +27,7 @@ export const DashboardToolbar = (props: DashboardToolbarProps) => {
   const { dashboardName } = props;
 
   const { isEditMode, setEditMode } = useEditMode();
-  const { addPanelGroup, addPanel, reset, save } = useDashboardActions();
+  const { openAddPanelGroup, openAddPanel, reset, save } = useDashboardActions();
```

Comment:
I'm curious if any of these count as a BREAKINGCHANGE in the changelog?

Reply:
i was going to point this out too. i think it is a break change every time we change the dashboard-provider-api

Reply:
Yeah, unfortunately it is. 😢  I think this API should be stable very soon though. I'm working on another PR at the moment to remove a bunch of "public methods" on the store API which should shrink the surface are of what's public quite a bit.